test/HorizontalMultivecTests.js

```diff
@@ -0,0 +1,269 @@
+/* eslint-env node, jasmine */
+import {
+  configure,
+  // render,
+} from 'enzyme';
+
+import Adapter from 'enzyme-adapter-react-16';
+
+// Utils
+import {
+  mountHGComponent,
+  removeHGComponent,
+} from '../app/scripts/utils';
+
+configure({ adapter: new Adapter() });
+
+describe('Horizontal heatmaps', () => {
+  let hgc = null;
+  let div = null;
+
+  beforeAll((done) => {
+    ([div, hgc] = mountHGComponent(div, hgc,
+      viewConf1,
+      done,
+      {
+        style: 'width:800px; height:400px; background-color: lightgreen',
+        bounded: true,
+      })
+    );
+  });
+
+  it('not have errors in the loaded viewconf', (done) => {
+    done();
```

Comment:
Where is the test? Am I missing something?

Reply:
The test is just loading that viewconf without throwing an error.

Reply:
Is there a way to explicitly test for no error?